core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-editor-modal.component.ts

```diff
@@ -120,6 +120,11 @@ export class QuestionSuggestionEditorModalComponent
       this.question, this.misconceptionsBySkill);
   }
 
+  getQuestionValidationErrorMessage(): string | null {
```

Comment:
Do we need to add a test case for this one?

Reply:
Yeah a smoke test was added since it just calls question validation service. More test cases were added in core/templates/services/question-validation.service.spec.ts,